ote/ote/utils/runners.py

```diff
@@ -79,3 +79,10 @@ def run_with_termination(cmd):
         stderr = stderr.decode('utf-8')
         print(stderr, end='')
         sys.stdout.flush()
+
+    time.sleep(1)
```

Comment:
May be something like ` dmesg -T | egrep -i 'killed process'` would allow to avoid waiting and accessing to the file system, but it needs to be tested.